### PR TITLE
OSDOCS-8806: cleaning and recreating custom certificates

### PR DESCRIPTION
**Pull request by aravipra**
Version(s):
4.16+

Issue:
[OSDOCS-8806](https://issues.redhat.com/browse/OSDOCS-8806)

Link to docs preview:
[Cleaning up and recreating the custom certificates](https://77156--ocpdocs-pr.netlify.app/microshift/latest/microshift_configuring/microshift-custom-ca.html#microshift-custom-ca-certificates-cleaning_microshift-custom-ca)

QE review:
- [x] QE has approved this change.

SME review:
- [x] SME has approved this change.

Additional information:
[Release notes PR](https://github.com/openshift/openshift-docs/pull/77217)

### CODE REVIEW

**Comment by ggiguash** [comment]
This is LGTM for me.

File: modules/microshift-custom-ca-cert-cleaning.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * microshift_security_compliance/microshift-custom-ca.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-custom-ca-certificates-cleaning_{context}"]
+= Cleaning and recreating the custom certificates
+
+To stop the {microshift-short} services, clean the {microshift-short} certificates and recreate the {microshift-short} certificates, use the following steps.
```
Comment (ShaunaDiaz) [inline modules/microshift-custom-ca-cert-cleaning.adoc:9]: What does "clean" mean? Are we removing previous certificates and their data? Be good say what and why and not use jargon.

Comment (aravipra) [reply modules/microshift-custom-ca-cert-cleaning.adoc]: Yes. Okay.
changing 'clean' with 'clean up'.

**Review by mletalie — COMMENTED** [review]
/label peer-review-in-progress

File: modules/microshift-custom-ca-cert-cleaning.adoc [hunk 2]
```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * microshift_security_compliance/microshift-custom-ca.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-custom-ca-certificates-cleaning_{context}"]
+= Cleaning up and recreating the custom certificates
+
+To stop the {microshift-short} services, clean up the {microshift-short} certificates and recreate the {microshift-short} certificates, use the following steps.
```
Comment (mletalie) [inline modules/microshift-custom-ca-cert-cleaning.adoc:9]: The title refers to cleaning up and recreating the custom certificates, but this lead sentence refers specifically to Microshift services and certificates (and addresses stopping) as well. 
Is it possible to make the title and lead sentence more in line with one another? 
Don't know enough about the feature but wanted to point that out regardless.

File: modules/microshift-custom-ca-cert-cleaning.adoc [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+MicroShift service was stopped
+Cleanup succeeded
+----
+
+. Restart the {microshift-short} services to recreate the {microshift-short} certificates by running the following command:
++
+[source,terminal]
+----
+$ sudo systemctl start microshift
+----
```
Comment (mletalie) [inline modules/microshift-custom-ca-cert-cleaning.adoc:34]: Should an example output be included here as well?

**Review by mletalie — COMMENTED** [review]
/label peer-review-done
/remove-label peer-review-in-progress
/remove-label peer-review-needed
/lgtm

File: modules/microshift-custom-ca-cert-cleaning.adoc (hunk 3 above)
Comment (aravipra) [reply modules/microshift-custom-ca-cert-cleaning.adoc:34]: No output is required as it is just starting the service that was stopped. The example output confirms the same.

File: modules/microshift-custom-ca-cert-cleaning.adoc (hunk 2 above)
Comment (aravipra) [reply modules/microshift-custom-ca-cert-cleaning.adoc:9]: I have replaced "Microshift" with "custom". This should be good.

**Pull request merged**